aten/src/TH/THStorageFunctions.cpp

```diff
@@ -39,10 +39,9 @@
 #include <TH/generic/THStorageCopy.cpp>
 #include <TH/THGenerateBFloat16Type.h>
 
-THStorage* THStorage_new(caffe2::TypeMeta data_type) {
+THStorage* THStorage_new() {
```

Comment:
Need to verify all callsites to `THStorage_new` eventually properly set data type

Reply:
One of the call sites is in `maybe_resize_storage_cpu` in `aten/src/ATen/native/Resize.h`, I'll add an assert after that call to make sure the dtype is correct.

The other call site is in `THStorage_(new)()` in `aten/src/TH/generic/THStorage.cpp`, which just returns the result of `THStorage_new()`. `THStorage_(new)()` has two call sites in `aten/src/TH/generic/THTensor.cpp`, and the type is explicitly set with `scalar_t` in those spots.